libs/journeys/ui/src/components/Video/Video.tsx

```diff
@@ -167,24 +177,34 @@ export function Video({
         }
       }}
     >
-      {playerRef.current != null && video != null && (
-        <VideoEvents
-          player={playerRef.current}
-          blockId={blockId}
-          videoTitle={video.title[0].value}
-          videoId={video.id}
-          startAt={startAt}
-          endAt={endAt}
-        />
-      )}
-      {video?.variant?.hls != null ? (
+      {playerRef.current != null &&
+        eventVideoTitle != null &&
+        eventVideoId != null && (
+          <VideoEvents
+            player={playerRef.current}
+            blockId={blockId}
+            videoTitle={eventVideoTitle}
+            videoId={eventVideoId}
+            startAt={startAt}
+            endAt={endAt}
+          />
+        )}
+      {videoId != null ? (
         <>
           <video
             ref={videoRef}
             className="video-js vjs-big-play-centered"
             playsInline
           >
-            <source src={video.variant.hls} type="application/x-mpegURL" />
+            {video?.variant?.hls != null && (
+              <source src={video.variant.hls} type="application/x-mpegURL" />
+            )}
+            {source === 'youTube' && (
+              <source
+                src={`https://www.youtube.com/watch?v=${videoId}`}
+                type="video/youtube"
+              />
+            )}
```

Comment:
Use the typescript enum type here just in case we ever change the value it will be easy to track.

```suggestion
            {source === VideoBlockSource.internal && video?.variant?.hls != null && (
              <source src={video.variant.hls} type="application/x-mpegURL" />
            )}
            {source === VideoBlockSource.youTube && (
              <source
                src={`https://www.youtube.com/watch?v=${videoId}`}
                type="video/youtube"
              />
            )}
```